build_packages.py

```diff
@@ -13,6 +14,10 @@
 from src.repository import Repository
 from src.version import FullVersion
 
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
```

Comment:
Quick question: is this specific line needed? As we don't use it in this script, I guess we don't need it, right?